unlock-app/src/components/creator/Dashboard.js

```diff
@@ -4,9 +4,15 @@ import React, { Component } from 'react'
 import Layout from '../interface/Layout'
 import CreatorAccount from './CreatorAccount'
 import CreatorLocks from './CreatorLocks'
+import {setTransaction} from '../../actions/transaction'
+import { connect } from 'react-redux'
 
-export default class Dashboard extends Component {
+export class Dashboard extends Component {
   render() {
+    if (!this.props.account) {
+      return null //loading
+    }
```

Comment:
We should probably eventually handle that at a higher level.. But that'll do for now!